app/helpers/masthead_helper.rb

```diff
@@ -28,6 +28,11 @@ def digital_collection_path
     search_catalog_path(facet_params)
   end
 
+  def iiif_item_path
+    facet_params = { f: { collection_type: ['Digital Collection'] } }
```

Comment:
Is that the right link? We're using this today in -prod:

```suggestion
    facet_params = { f: {iiif_resources: ['available'] }
```

Reply:
Thanks. I meant to put that but then must have gotten distracted.

Reply:
Thanks I meant to do that but got distracted